separate_proxy.yaml

```diff
@@ -0,0 +1,68 @@
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - snmp_types.yaml
+
+
+node_templates:
+
+  SNMPSecurityGroup:
```

Comment:
Node templates should not be CamelCased but instead lowercased with underscores
`snmp_security_group`
This applies to all node templates in both blueprints